tools/src/main/resources/create-sql-scripts.sh

```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env bash
+
+# Delete Existing new-install.sql
+rm upgrade/new-install.sql
+# Delete Existing upgrade.sql
+rm upgrade/upgrade.sql
 # Create an SQL script for a new installation
 
 # Turns off foreign key cheks to speed up script execution
-cat database/mysql_script_header.sql > upgrade/new-install.sql
+cat database/mysql_script_header.sql >> upgrade/new-install.sql
```

Comment:
The purpose of the single > is to overwrite any contents of the file that may exist, this removed the need delete a new file. 

Also the >> may not create a new file depending on the permissions